[2022-08-06 19:38:56,224] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 19:38:56,252] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 19:38:56,252] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:56,252] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:56,252] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:56,278] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-12 00:00:00+00:00
[2022-08-06 19:38:56,290] {standard_task_runner.py:52} INFO - Started process 2814 to run task
[2022-08-06 19:38:56,294] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-12T00:00:00+00:00', '--job-id', '866', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmplpscxogp', '--error-file', '/tmp/tmpedsc4wsj']
[2022-08-06 19:38:56,295] {standard_task_runner.py:80} INFO - Job 866: Subtask second_task
[2022-08-06 19:38:56,382] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:56,483] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-12T00:00:00+00:00
[2022-08-06 19:38:56,484] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:56,484] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:56,499] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:56,501] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:56,502] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:56,538] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220512T000000, start_date=20220806T193856, end_date=20220806T193856
[2022-08-06 19:38:56,588] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:56,665] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:46,844] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 21:52:46,864] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 21:52:46,864] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:46,864] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:46,864] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:46,885] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-12 00:00:00+00:00
[2022-08-06 21:52:46,892] {standard_task_runner.py:52} INFO - Started process 2996 to run task
[2022-08-06 21:52:46,895] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-12T00:00:00+00:00', '--job-id', '866', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpfvg5bu2f', '--error-file', '/tmp/tmpky39itm7']
[2022-08-06 21:52:46,896] {standard_task_runner.py:80} INFO - Job 866: Subtask second_task
[2022-08-06 21:52:46,975] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:47,086] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-12T00:00:00+00:00
[2022-08-06 21:52:47,087] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:47,087] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:47,102] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:47,104] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:47,104] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:47,139] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220512T000000, start_date=20220806T215246, end_date=20220806T215247
[2022-08-06 21:52:47,227] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:47,266] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:14,615] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 21:58:14,638] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 21:58:14,639] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:14,639] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:14,640] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:14,673] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-12 00:00:00+00:00
[2022-08-06 21:58:14,681] {standard_task_runner.py:52} INFO - Started process 2804 to run task
[2022-08-06 21:58:14,691] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-12T00:00:00+00:00', '--job-id', '867', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpnsyr5fxp', '--error-file', '/tmp/tmpmgafxy7b']
[2022-08-06 21:58:14,694] {standard_task_runner.py:80} INFO - Job 867: Subtask second_task
[2022-08-06 21:58:14,838] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:14,950] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-12T00:00:00+00:00
[2022-08-06 21:58:14,951] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:14,952] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:14,971] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:14,974] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:14,974] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:15,022] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220512T000000, start_date=20220806T215814, end_date=20220806T215815
[2022-08-06 21:58:15,071] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:15,135] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:14,246] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 22:13:14,260] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 22:13:14,260] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:14,260] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:14,260] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:14,278] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-12 00:00:00+00:00
[2022-08-06 22:13:14,286] {standard_task_runner.py:52} INFO - Started process 2800 to run task
[2022-08-06 22:13:14,289] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-12T00:00:00+00:00', '--job-id', '863', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpz9ssyr75', '--error-file', '/tmp/tmpfnrq_3l6']
[2022-08-06 22:13:14,290] {standard_task_runner.py:80} INFO - Job 863: Subtask second_task
[2022-08-06 22:13:14,378] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:14,495] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-12T00:00:00+00:00
[2022-08-06 22:13:14,496] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:14,496] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:14,512] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:14,514] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:14,514] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:14,609] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220512T000000, start_date=20220806T221314, end_date=20220806T221314
[2022-08-06 22:13:14,662] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:14,712] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:31,644] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 22:24:31,660] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2022-08-06 22:24:31,660] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:31,660] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:31,660] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:31,679] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-12 00:00:00+00:00
[2022-08-06 22:24:31,686] {standard_task_runner.py:52} INFO - Started process 2855 to run task
[2022-08-06 22:24:31,689] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-12T00:00:00+00:00', '--job-id', '866', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpeqczgbzz', '--error-file', '/tmp/tmpsdzr07l4']
[2022-08-06 22:24:31,690] {standard_task_runner.py:80} INFO - Job 866: Subtask second_task
[2022-08-06 22:24:31,779] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-12T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:31,865] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-12T00:00:00+00:00
[2022-08-06 22:24:31,866] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:31,867] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:31,880] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:31,881] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:31,881] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:31,909] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220512T000000, start_date=20220806T222431, end_date=20220806T222431
[2022-08-06 22:24:31,941] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:31,971] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
